scripts/services.js

```diff
@@ -1,4 +1,6 @@
 import createJsonRequest from "./utils.js";
+import {tableHeader} from "./utils.js";
+import {tableContent} from "./utils.js";
```

Comment:
multiple imports of same file/ class is not recommended

Reply:
Grouped import statements in the latest commit.